types/fundamental/real32quant/README.md

```diff
@@ -0,0 +1,20 @@
+# Real Column Types with variable width
+
+## Fields
+
+ * `FloatReal32Quant2_-1_1`
+ * `DoubleReal32Quant2_-1_1`
```

Comment:
The Binary Format Specification says we support a minimum bit width of 1 up to 32. We should exercise these two extremes.